wiki/Tournaments/OMIC/2019_4K/en.md

```diff
@@ -2,9 +2,415 @@
+
+| Sunday, 28 July 2019 |  |  |  |  |
+| --: | :-: | :-: | :-- | :-- |
+| [alvinrainbow](https://osu.ppy.sh/users/4783042) ::{ flag=ID }:: | 0 | **5** | ::{ flag=ID }:: **[Yangga](https://osu.ppy.sh/users/12143152)** | [#1](https://osu.ppy.sh/community/matches/53655829) |
+| **[Proxae](https://osu.ppy.sh/users/8410575)** ::{ flag=ID }:: | **5** | 0 | ::{ flag=ID }:: [Merlinch](https://osu.ppy.sh/users/4901971) | -win by default- |
+| [Yamada Kitaro](https://osu.ppy.sh/users/7670805) ::{ flag=ID }:: | 0 | **5** | ::{ flag=ID }:: **[-\[ChoKy\]-](https://osu.ppy.sh/users/4691146)** | [#1](https://osu.ppy.sh/community/matches/53658604) |
+| **[Fyl](https://osu.ppy.sh/users/10069307)** ::{ flag=ID }:: | **5** | 2 | ::{ flag=ID }:: [2ndlegend](https://osu.ppy.sh/users/7621604) | [#1](https://osu.ppy.sh/community/matches/53661646) |
+| [Artem1s](https://osu.ppy.sh/users/12561379) ::{ flag=ID }:: | 0 | **5** | ::{ flag=ID }:: **[Slowless](https://osu.ppy.sh/users/10053987)** | -win by default- |
+
+### Round of 16
+
+| Saturday, 20 July 2019 |  |  |  |  |
+| --: | :-: | :-: | :-- | :-- |
+| **[Kevin-sama](https://osu.ppy.sh/users/6991664)** ::{ flag=ID }:: | **5** | 1 | ::{ flag=ID }:: [Fyl](https://osu.ppy.sh/users/10069307) | [#1](https://osu.ppy.sh/community/matches/53470726) |
+| **[Heriza-](https://osu.ppy.sh/users/8911771)** ::{ flag=ID }:: | **5** | 3 | ::{ flag=ID }:: [Yangga](https://osu.ppy.sh/users/12143152) | [#1](https://osu.ppy.sh/community/matches/53474229) |
+| **[ChocoCoki-](https://osu.ppy.sh/users/3565870)** ::{ flag=ID }:: | **5** | 0 | ::{ flag=ID }:: [2ndlegend](https://osu.ppy.sh/users/7621604) | [#1](https://osu.ppy.sh/community/matches/53475726) |
+
+| Sunday, 21 July 2019 |  |  |  |  |
+| --: | :-: | :-: | :-- | :-- |
+| [Merlinch](https://osu.ppy.sh/users/4901971) ::{ flag=ID }:: | 1 | **5** | ::{ flag=ID }:: **[\_claidheamh](https://osu.ppy.sh/users/4686520)** | [#1](https://osu.ppy.sh/community/matches/53491947) |
+| **[\[KonG-69\]](https://osu.ppy.sh/users/8847412)** ::{ flag=ID }:: | **5** | 2 | ::{ flag=ID }:: [alvinrainbow](https://osu.ppy.sh/users/4783042) | [#1](https://osu.ppy.sh/community/matches/53493068) |
+| [Artem1s](https://osu.ppy.sh/users/12561379) ::{ flag=ID }:: | 1 | **5** | ::{ flag=ID }:: **[Yamada Kitaro](https://osu.ppy.sh/users/7670805)** | [-win by default-](https://osu.ppy.sh/community/matches/53494572) |
+| [Slowless](https://osu.ppy.sh/users/10053987) ::{ flag=ID }:: | 1 | **5** | ::{ flag=ID }:: **[-\[ChoKy\]-](https://osu.ppy.sh/users/4691146)** | [#1](https://osu.ppy.sh/community/matches/53497372) |
+| [Proxae](https://osu.ppy.sh/users/8410575) ::{ flag=ID }:: | 0 | **5** | ::{ flag=ID }:: **[reyss](https://osu.ppy.sh/users/4557440)** | [#1](https://osu.ppy.sh/community/matches/53499829) |
+
+### Group Distribution 
+
+| Saturday, 6 July 2019 |  |  |  |  |
```

Comment:
please look at the [template](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/TEMPLATE.md?plain=1#L125-L132)